src/codegen/codegen_neuron_cpp_visitor.cpp

```diff
@@ -31,6 +31,7 @@ using visitor::RenameVisitor;
 using visitor::VarUsageVisitor;
 
 using symtab::syminfo::NmodlType;
+using ParamVector = std::vector<std::tuple<std::string, std::string, std::string, std::string>>;
```

Comment:
Please remove this. `ParamVector` should be correctly defined in a header.

Reply:
For some reason, when I remove this, or even just put it in the header file only, the compilation fails (clang 15.0.0 on MacOS), even though it should be visible due to this snippet:
https://github.com/BlueBrain/nmodl/blob/5366532ee2ca2f4961e635672f18043c69e23443/src/codegen/codegen_cpp_visitor.hpp#L266

Anything obvious that I'm missing here?